gadgets/trace_exec/gadget.yaml

```diff
@@ -20,6 +20,9 @@ structs:
     - name: comm
       attributes:
         template: comm
+    - name: pcomm
```

Comment:
Please add description to specify it's the one from the parent. Otherwise, `comm` and `pcomm` will have the same description (coming from the `comm` template).